lib/blpapi-wrapper.ts

```diff
@@ -76,8 +76,8 @@ var EVENT_TYPE = {
 var REQUEST_TO_RESPONSE_MAP: { [index: string]: string; } = {
     // //blp/refdata
     'HistoricalDataRequest': 'HistoricalDataResponse',
-    'IntraDayTickRequest':   'IntraDayTickResponse',
-    'IntraDayBarRequest':    'IntraDayBarResponse',
+    'IntradayTickRequest':   'IntradayTickResponse',  
```

Comment:
Please remove the trailing whitespace here; it is being caught by `tslint` when TravisCI kicks-off.  You can also run `make check` to run exactly what our CI process executes.